### DISCUSSION TITLE
Does copilot works only in the official VSCode for linux?

### DISCUSSION THREAD

**Discussion by nullndr** [community/community#13771] [Copilot Conversations] [answered]
I've obtained access to the preview of copilot, and I wanted to install it from the marketplace, but the marketplace cannot find it.

Strange, I downloaded the extension from the microsoft site, and moved it into `$HOME/.vscode-oss/extensions`. The extension appeared, I was able to login ecc, but now the extension show a banner with the following text:

```
GitHub Copilot requires VS Code 1.57+. Please update your VS Code
```

My visual studio code oss is `1.65.2`, installed from the official Arch Linux [repository](https://archlinux.org/packages/?name=code).

**Comment by davecheney** [comment, marked as answer]
Yes, only VSCode is supported. OSSCode or VSCodium are not supported.

**Reply by joaojhgs** [reply]
I'm using the official VScode for linux but it still does not authenticates.

**Reply by nullndr** [reply]
Do you know if it will be supported?

**Reply by ImUrX** [reply]
why not? it's not really open-sourcey to do that lol
